src/web/components/SharingPermission/SharingPermissionsTable.spec.tsx

```diff
@@ -28,11 +28,11 @@ describe('SharingPermissionsTable', () => {
     expect(searchInput).toHaveValue('Participant 1');
   });
 
-  it('renders delete permissions button when there are permission select', () => {
+  it('renders delete permissions button when there are permissions selected', () => {
     render(<SharedWithParticipants />);
 
-    const firstParticipantItem = screen.getByText('Participant 1');
-    fireEvent.click(firstParticipantItem);
+    const firstCheckbox = screen.getAllByRole('checkbox')[0];
```

Comment:
The row in the table no longer triggers the `onClick`, the user must click the checkbox instead. Here I am just grabbing the first one, which in this case will be the 'Select all' checkbox.